hvpy/__init__.py

```diff
@@ -1 +1,2 @@
+from .facade import getJP2Image
```

Comment:
With this I am able to do
```py
from hvpy import getJP2Image

date_obj = datetime(2022, 1, 1, 23, 59, 59)
params = {"date": date_obj, "sourceId": 14, "jpip": True, "json": False}
result = getJP2Image(**params)
```
But this is not a good way to pass the parameters, right?

Reply:
Check out this question https://stackoverflow.com/questions/2521901/get-a-list-tuple-dict-of-the-arguments-passed-to-a-function

Reply:
I am ok with that being the way we test, but for examples, we should just pass them as keywords.